mungegithub/mungers/approval-handler.go

```diff
@@ -90,8 +90,31 @@ func (h *ApprovalHandler) Munge(obj *github.MungeObject) {
 	}
 
 	approverSet := sets.String{}
+	if obj.Issue.User.Name != nil {
+		approverSet.Insert(*obj.Issue.User.Name)
```

Comment:
@fejta Automatically adding author to the `approverSet` should be the simplest way to make the process simple when you modify your own code/area.